dpnp/dpnp_iface_mathematical.py

```diff
@@ -1325,18 +1326,35 @@ def negative(x1, **kwargs):
     return call_origin(numpy.negative, x1, **kwargs)
 
 
-def power(x1, x2, dtype=None, out=None, where=True, **kwargs):
+def power(x1,
+          x2,
+          /,
+          out=None,
+          *,
+          where=True,
+          dtype=None,
+          subok=True,
+          **kwargs):
     """
     First array elements raised to powers from second array, element-wise.
 
+    An integer type (of either negative or positive value, but not zero)
+    raised to a negative integer power will return an array of zeroes.
+
     For full documentation refer to :obj:`numpy.power`.
 
+    Returns
+    -------
+    y : dpnp.ndarray
+        The bases in `x1` raised to the exponents in `x2`.
+    
     Limitations
     -----------
-    Parameters ``x1`` and ``x2`` are supported as either :obj:`dpnp.ndarray` or scalar.
-    Parameters ``dtype``, ``out`` and ``where`` are supported with their default values.
+    Parameters `x1` and `x2` are supported as either :class:`dpnp.ndarray` or scalar,
```

Comment:
Parameters `x1` and `x2` are supported as either :class:`dpnp.ndarray`,  :class:`dpctl.tensor.usm_ndarray`, or scalar,